src/web/app/pages-instructor/instructor-student-records-page/instructor-student-records-page.component.ts

```diff
@@ -188,4 +201,89 @@ export class InstructorStudentRecordsPageComponent implements OnInit {
       isEditing: false,
     };
   }
+
+  /**
+   * Deletes an instructor comment.
+   */
+  deleteComment(data: { responseId: string, index: number}): void {
```

Comment:
Are you copying from the result page? I think most of the comment related features are the same. Can we extract to a common parent class?

Reply:
>Are you copying from the result page?

Yeap

>Can we extract to a common parent class?

Sounds good, I'll put it in the root of `pages-instructor`?

Reply:
Actually, I will hold off on this until https://github.com/TEAMMATES/teammates/pull/10136 is resolved since there's quite an extensive refactoring going on there that would affect how we extract the methods here